src/Documentation/sidebar.json

```diff
@@ -82,6 +82,26 @@
     "slug": "user-guide",
     "source": "user-guide/index.md",
     "children": [
+      {
+        "label": "Installation",
+        "slug": "install",
+        "source": "install/index.md",
+        "children": [
+          {
+            "label": "MacOS",
+            "slug": "macos"
+          },
+          "windows",
+          "debian",
+          "fedora",
+          "arch",
```

Comment:
These 3 may need labels: `Debian/ Ubuntu`, `Fedore/ CentOS`, `Arch Linux` to match https://dvc-org-pr-695.herokuapp.com/doc/user-guide/install.

Reply:
And/or actually may need to merge them per https://github.com/iterative/dvc.org/pull/695#pullrequestreview-301594486?

> Linux - seems we can merge into one page with H2 (##) titles...